e2e/tests/test_e2e.py

```diff
@@ -118,6 +118,61 @@ def test_enable_load_balancer(self):
         self.assertEqual(repl_svc_type, 'ClusterIP',
                          "Expected ClusterIP service type for replica, found {}".format(repl_svc_type))
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_lazy_spilo_update(self):
+        '''
+        Test lazy update for the Spilo image: operator changes a stateful set but lets pods run with the old image 
+        until they are recreated for reasons other than operator's activity. That works because the operator uses 
+        "onDelete" pod update policy for stateful sets.
+
+        The test covers:
+        1) enabling lazy upgrade in existing operator deployment
+        2) forcing the normal rolling upgrade by changing the operator configmap and restarting its pod
+        '''
+
+        k8s = self.k8s
+        pod0 = "acid-minimal-cluster-0"
+        pod1 = "acid-minimal-cluster-1"
```

Comment:
hm, maybe we should use a constant for this minimal cluster, because we hard code it in so many tests. Then I would simply pass CONSTANT+0/1 to the other methods instead of defining variables, but that's just my taste

Reply:
let's leave it as is right now and instead start a separate issue for tidying up e2e tests. I want to avoid cluttering this important PR with unrelated changes